lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -59,7 +60,7 @@ def initialize(metadata)
           @modified_at = Time.now
         end
 
-        attr_reader :unique_id, :metadata, :created_at, :modified_at, :state
+        attr_reader :unique_id, :chunk_id, :metadata, :created_at, :modified_at, :state
```

Comment:
It is added to use in `OutSecondary` ( https://github.com/fluent/fluentd/pull/1154/files#diff-68f04670661c50196427de0d60c06636R84 )